### PR TITLE
Feature/23 create fe master page

### PR DESCRIPTION
**Pull request by harryul**
Coding header, side bar for master page

### CODE REVIEW

File: src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             if (!optionsBuilder.IsConfigured)
             {
-               optionsBuilder.UseSqlServer("Data Source=5CD1199VTK;Initial Catalog=PatientCheckIn;Integrated Security=True");
+               optionsBuilder.UseSqlServer("Data Source=5CD1199VDQ;Initial Catalog=PatientCheckIn;Integrated Security=True");
             }
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs]: Connection strings should be retrieved from appsettings.

File: src/PatientChecking/PatientChecking/Startup.cs [hunk 2]
```diff
@@ -60,7 +60,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseSwaggerUI(c =>
             {
                 c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
-                c.RoutePrefix = string.Empty;
+                //c.RoutePrefix = string.Empty;
             });
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/Startup.cs]: Please remove all comments that are not intended to let reader to be aware of.

File: src/PatientChecking/PatientChecking/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -1,48 +1,77 @@
...
-<html lang="en">
+<html lang="en" dir="ltr">
 <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - PatientChecking</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <title>@ViewData["Title"] | PatientChecking</title>
     <link rel="stylesheet" href="~/css/site.css" />
+    <script src="https://code.jquery.com/jquery-3.4.1.js"></script>
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/Views/Shared/_Layout.cshtml]: please add js files as part of the project so the cshtml can refer them as local assets.

Comment (harryul) [reply src/PatientChecking/PatientChecking/Views/Shared/_Layout.cshtml]: Js files have been added to project as local assets

File: src/PatientChecking/PatientChecking/Startup.cs (hunk 2 above)
Comment (harryul) [reply src/PatientChecking/PatientChecking/Startup.cs]: comments have been removed

File: src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs (hunk 1 above)
Comment (harryul) [reply src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs]: Connnection string has been defined in appsettings file and used in startup

File: src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs [hunk 4]
```diff
@@ -25,10 +25,7 @@ public PatientCheckInContext(DbContextOptions<PatientCheckInContext> options)
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            if (!optionsBuilder.IsConfigured)
-            {
-               optionsBuilder.UseSqlServer("Data Source=5CD1199VTK;Initial Catalog=PatientCheckIn;Integrated Security=True");
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs:30]: don't use PC name "5CD1199VTK", you can use Data Source=(local) so all local machines can use it

File: src/PatientChecking/PatientChecking/appsettings.json [hunk 5]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Data Source=5CD1199VDQ;Initial Catalog=PatientCheckIn;Integrated Security=True"
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/appsettings.json]: use local data source like suggestion above.

File: src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs (hunk 4 above)
Comment (harryul) [reply src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs:30]: fixed as per reviewer request

File: src/PatientChecking/PatientChecking/appsettings.json (hunk 5 above)
Comment (harryul) [reply src/PatientChecking/PatientChecking/appsettings.json]: fixed as per reviewer request

**Pull request merged**